src/enums/request-code.ts

```diff
@@ -1,4 +1,4 @@
-/** Enum for Request (action) codes. Not to be confused with NR Action codes. */
```

Comment:
I think this was added because `request` alone is vague as there is request type and request action etc..
No biggie, just some thoughts.

Reply:
The comment? Yes, I added it previously because I found the naming confusing. Is this OK or should I go back to being explicit? (I think it's much more clear now with the enums anyway.)

Reply:
For sure! I'm fine with this, was just sharing what i encountered when working in Namex UI :)

Reply:
Is there something (easy) that I can do to make this sync up with Namex better?

Reply:
This make sense for this UI. 
It's on Namex side where it gets confusing since they use request type and request action.
These do align with request action but i don't think we need to be explicit here. 
Either way, its allll good :)